lib/model/init.js

```diff
@@ -0,0 +1,46 @@
+const inquirer = require('inquirer');
```

Comment:
The code provided in this file is still the CLI part. It is better to put this file as lib/cli/input/init.js. The code structure would be as follows.
```
lib/
    controller/
        init.js
    cli/
        input/
            init.js
        output/
             @init.js
```
For each command put in cli/input, there would be a matching prompt command. This structure should be completely testable. If you can propose a different structure that is more easier to test, I am willing to consider that too.

Reply:
I agree with the structure as it makes it easier to test.